samples/certs/Makefile

```diff
@@ -121,3 +133,52 @@ root-key.pem:
 	@echo "O = $(CITADEL_ORG)" >> $@
 	@echo "CN = $(CITADEL_CN)" >> $@
 	@echo "L = $(L:/=)" >> $@
+
+#------------------------------------------------------------------------
+##WORKLOAD_CERTS=1 <name>-certs: generate workload certificates using above intermediate ca Includes all PEM needed.
+
+%/workload-cert-chain.pem: %/workload-cert.pem %/ca-cert.pem root-cert.pem
+	@echo "generating $@"
+	@cat $^ > $@
+
+%/workload-cert.pem: %/workload.csr
+	@echo "generating $@"
+	@openssl x509 -req -days $(WORKLOAD_DAYS) \
+		-CA $(dir $<)/ca-cert.pem  -CAkey $(dir $<)/ca-key.pem -set_serial $(CITADEL_SERIAL) \
+		-extensions req_ext -extfile $(dir $<)/workload.conf \
+		-in $< -out $@
+
+%/workload.csr: L=$(dir $@)
```

Comment:
what is the purpose of L=$(dir $@)? You have duplicate targets in this makefile, which typically isn't idiomatic in make. You can combine these into one on line 152 IIRC.